src/prefect/cli/__init__.py

```diff
@@ -48,6 +48,7 @@ def cli():
     Miscellaneous Commands:
         version     Get your current Prefect version
         config      Output your Prefect config
```

Comment:
Not related to your PR but I'm just noticing the word "your" here which should be removed -- "Output the Prefect Config", "Print the Prefect version"